docu/files/35_wherenext.html.eco

```diff
@@ -6,4 +6,8 @@ title: 'Where Do I Go Next?'
 <p>For example, if you are a developer, you can explore the&nbsp;<a href="https://devportal.yaas.io/">Dev Portal</a>. You can try our&nbsp;<a href="https://devportal.yaas.io/gettingstarted/">Getting Started Guides</a>. If you are interested in a particular solution, such as&nbsp;SAP Hybris Profile, the&nbsp;<a href="https://devportal.yaas.io/solutions/">Solutions</a>&nbsp;section of the DevPortal is what you are looking for. Finally, if you prefer reading documentation, have a look at the&nbsp;<a href="https://devportal.yaas.io/services/">API Docs</a>&nbsp;section where every service published on the YaaS Market is thoroughly documented. Most documents include tutorials, so you can try out the service as you read through the documentation.</p>
 <p>If you are not a developer, then the&nbsp;<a href="https://knowledge.yaas.io/">Knowledge Hub</a>&nbsp;is the right place for you. As you have already seen in this section, the training provides you with free online training on various YaaS topics.&nbsp;We provide you with the&nbsp;guides that help you configure and use your packages efficiently. Also,&nbsp;you can get recommendations, examples of common pitfalls and best approaches, news, and more.</p>
 <p>Finally, if you are interested in using YaaS to build on your&nbsp;existing SAP portfolio, you can use guides and articles in&nbsp;the Knowledge Hub or look on the DevPortal for the technical details.</p>
+<p>Beside resources available on the Knowledge Hub and Dev Portal, you might be interested in the training offerings from SAP Hybris Education:</p>
```

Comment:
<p>In addition to the resources available on the Knowledge Hub and the Dev Portal, you might be interested in these training offerings from SAP Hybris Education:</p>